migrations/20170216104317_create_audit_log_for_trial_deduplication.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+exports.up = (knex) => (
+  knex.schema
+    .createTable('trial_deduplication_logs', (table) => {
+      table.uuid('record_id')
+        .primary();
```

Comment:
`record_id` can't be the primary key, otherwise it won't be possible to implement the pattern described on https://github.com/opentrials/opentrials/issues/676#issuecomment-280060487. More specifically, allowing us to know not only what was the latest match a record had, but the previous ones as well. For example:

trial_id | record_id | method | commit | created_at | updated_at
------------ | -------------|------------ | -------------|------------ | -------------
A | B|title | | 2017-02-01 | |
A | B|identifier |  | 2017-02-05 | |

Maybe not, but I'd imagine it would require a separate `id` field (can be a simple auto-incrementing int).